posix.mak

```diff
@@ -13,6 +13,8 @@
 #
 # make BUILD=debug unittest => builds all unittests (for debug) and runs them
 #
+# make DEBUGGER=ddd std/XXXXX.debug => builds the module XXXXX and executes it in the debugger ddd
```

Comment:
How about adding a default value for `DEBUGGER` (`gdb` would probably be a sensible default)? You will probably need to pass it onward in the recursive make invocation.